### PR TITLE
Updated transaction id

### PR DESCRIPTION
**Pull request by dgarcia360**
- Moved embedded id outside meta to match other collections - @Vektrat @martinayora please adapt rest

### CODE REVIEW

File: spec/core/transaction/schemas/EmbeddedTransactionInfoDTO.yml [hunk 1]
```diff
@@ -1,8 +1,12 @@
 type: object
 required:
+  - id
   - meta
   - transaction
 properties:
+  id:
+    type: string
+    description: Internal resource identifier.
```
Comment (rg911) [inline spec/core/transaction/schemas/EmbeddedTransactionInfoDTO.yml:9]: Database record id ?

Comment (fboucquez) [reply spec/core/transaction/schemas/EmbeddedTransactionInfoDTO.yml:9]: @Vektrat , could you confirm if embedded transaction id are / are going to be outside metadata?

Comment (Vektrat) [reply spec/core/transaction/schemas/EmbeddedTransactionInfoDTO.yml:9]: they are yes, no id should be inside meta

**Comment by fboucquez** [comment]
Missing anyOf EmbeddedTransactionMetaDTO at https://github.com/nemtech/symbol-openapi/blob/master/spec/core/transaction/schemas/TransactionInfoDTO.yml meta?

If TransactionInfoDTO.yml accepts embedded transactions, it should accept the meta of embedded transactions.

**Comment by dgarcia360** [comment]
@fboucquez yes, just pushed this here https://github.com/nemtech/symbol-openapi/commit/aba129d5aadbe2572bba534d951d4d61e51b3e83

**Pull request merged**